api/telemetry.go

```diff
@@ -119,12 +130,17 @@ func (e *EverestServer) collectMetrics(ctx context.Context, url string) error {
 
 		for _, cl := range clusters.Items {
 			types[string(cl.Spec.Engine.Type)]++
+
+			// Number of backup schedules per DB cluster.
+			metrics = append(metrics, Metric{
+				Key:   fmt.Sprintf(telemetryBackupSchedulesKeyTemplate, string(cl.Spec.Engine.Type), cl.UID),
+				Value: strconv.Itoa(len(cl.Spec.Backup.Schedules)),
+			})
```

Comment:
Time-series databases don't work well with the unique keys. From prometheus [docs](https://prometheus.io/docs/practices/naming/):

> CAUTION: Remember that every unique combination of key-value label pairs represents a new time series, which can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

I think we should look for alternative approaches

Reply:
I don't think we have other options. We have only a key and a value and we need to uniquely identify a db.

Reply:
I see. I was just wondering if the goal can be achieved without using UIDs. What is the goal of the metric? I mean, why do we collect this information? 
Such design of the keys may also make it hard to configure the dashboards, is Peter (or whoever will configure the dashboards) ok with such a format?

Reply:
Per agreement, the metric has been removed.